packages/next/src/server/app-render/app-render.tsx

```diff
@@ -1163,6 +1179,7 @@ async function renderToHTMLOrFlightImpl(
                 stream: await continueDynamicPrerender(result.stream, {
                   getServerInsertedHTML,
                 }),
+                flightRenderResult,
               }
             } else {
               // This is the Static case
```

Comment:
This branch is the only fully static branch for PPR staticGeneration. This is where we should set metadata.flightData. The other branches imply dynamic RSC which means that on resumes we need to regenerate the RSC result and don't need to store a static .rsc response